android/databases/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.coinbase.wallet.libraries.databases">
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
I don't think the database need to read/write external storage.  We should be able to remove these permissions.

Reply:
removed